examples/images_contours_and_fields/image_antialiasing.py

```diff
@@ -72,6 +74,73 @@
     ax.set_title(f"interpolation='{interp}'")
 plt.show()
 
+###############################################################################
+# Data antialiasing and RGBA antialiasing
```

Comment:
I'd name this "Antialiasing colormapped data" and add a same-level "Antialiasing grayscale images".

And make some subsubsections here for "Data antialiasing" and "RGBA antialiasing".

// Well technically, grayscale is also a colormapping. The real difference is whether the colormapping is a straight line in RGB space or not. But I wouldn't go into the details here. Almost all colorful maps have quite some curve in RGB space.

Reply:
I'm not quite sure what you mean here?  The first section was all about data antialiasing, with black and white chosen to make it obvious what was going one.  This section is pointing out that this antialiasing is not the same as visual anti-aliasing.